Bottom: f8f35077a4f7494c80c9a32ea5a4231e57799a4f
Top:    6e65e6614b1bca6bcea374b98a30090f0b3ddaa7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-09-09 12:36:40 +0100

TEMPORARY: Comment out some DrawMark calls which might be out of context

Should fix this properly with notificatons to the GUI


---

diff --git a/src/action.c b/src/action.c
index 32e294c..edd79ba 100644
--- a/src/action.c
+++ b/src/action.c
@@ -4021,7 +4021,7 @@ ActionMarkCrosshair (int argc, char **argv, int x, int y)
     {
       if (Marked.status)
 	{
-	  DrawMark (true);
+//	  DrawMark (true);
 	  Marked.status = false;
 	}
       else
@@ -4029,16 +4029,16 @@ ActionMarkCrosshair (int argc, char **argv, int x, int y)
 	  Marked.status = true;
 	  Marked.X = Crosshair.X;
 	  Marked.Y = Crosshair.Y;
-	  DrawMark (false);
+//	  DrawMark (false);
 	}
     }
   else if (GetFunctionID (function) == F_Center)
     {
-      DrawMark (true);
+//      DrawMark (true);
       Marked.status = true;
       Marked.X = Crosshair.X;
       Marked.Y = Crosshair.Y;
-      DrawMark (false);
+//      DrawMark (false);
     }
   return 0;
 }
